packages/hash/api/src/graphql/resolvers/index.ts

```diff
@@ -279,25 +253,25 @@ export const resolvers = {
 
   // New knowledge field resolvers
 
-  PersistedEntity: {
+  EntityWithMetadata: {
     /**
-     * Determines whether a `PersistedEntity` instance should be treated as a
+     * Determines whether a `EntityWithMetadata` instance should be treated as a
```

Comment:
```suggestion
     * Determines whether an `EntityWithMetadata` instance should be treated as a
```

Reply:
Do we still need this thing anyway?

Reply:
Unfortunately yes, until we move to subgraphs exclusively on the frontend, this needs to be there for the different implementers of the Entity interface